components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -396,12 +396,13 @@ class GeckoEngineSession(
             }
         }
 
-        override fun onPageStop(session: GeckoSession, success: Boolean) {
+        override fun onPageStop(session: GeckoSession, success: Boolean, elapsed: Int) {
             // by the time we reach here, any new request will come from web content.
             // If it comes from the chrome, loadUrl(url) or loadData(string) will set it to
             // false.
             requestFromWebContent = true
             notifyObservers {
+                onElapsedTimeMS(elapsed)
```

Comment:
Naming nit: Let's not encode the unit (`ms`) into the method name, also this should probably say something about `load` in the name? 🤔

Reply:
I did not include the unit in the original underlying code for this and submitted it for review. Ehsan requested that I encode the unit in the name and I actually thought that was a good idea. I definitely agree that we should rename the method to include something about `load` and I will do that. I propose `onElapsedLoadTimeMS`?

Reply:
Check the new version of the patch!